docs/update_repo.md

```diff
@@ -0,0 +1,29 @@
+# How to Update Repository
+
+This repository is a branded version of the [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) and requires some consideration when updating.
+The basic update is to merge in upstream changes from a target release version and then apply branding. However, new roles may need additional testing added to them.
```

Comment:
```suggestion
The basic update is to merge in upstream changes from a target release version and then apply branding.
```

We dropped the extra instructions for that below, so let's drop it here too.

If we ever end up needing it, we can re-consider.